openstack/dcs/v2/instances/requests.go

```diff
@@ -0,0 +1,186 @@
+package instances
+
+import (
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/dcs/v2/tags"
```

Comment:
can `github.com/chnsz/golangsdk/common/tags` meet your needs？

Reply:
The `value` in the API cannot be omitted, so the common tag cannot be used in DCS.